src/core/lib/draw-layers.js

```diff
@@ -197,18 +197,18 @@ function drawLayersInViewport(gl, {
       );
 
       // Blend parameters must not be overriden
-      parameters = Object.assign({viewport: glViewport}, layer.props.parameters || {}, parameters);
+      let layerParameters = Object.assign({viewport: glViewport}, layer.props.parameters || {});
```

Comment:
@ibgreen not using previous layer's params (`parameters` -> `let layerParameters`) didn't fix this issue but made this change based on your comment in the issue.

Reply:
> not using previous layer's params (parameters -> let layerParameters) didn't fix this issue but made this change based on your comment in the issue.

Yes good. Perhaps there is one more bug, but your change here is correct.